src/Adapter/Configuration/LogsConfiguration.php

```diff
@@ -62,6 +63,17 @@ public function getConfiguration()
     public function updateConfiguration(array $configuration)
     {
         if ($this->validateConfiguration($configuration)) {
+            $checkEmails = explode(',', $configuration['logs_email_receivers']);
+            $errors = [];
+            foreach ($checkEmails as $email) {
+                if (!Validate::isEmail($email)) {
+                    $errors[] = $email;
```

Comment:
The error message has to be built here and not in the controllers.
Otherwise if I extend the symfony service and add a new check, I will have a "Invalid email address' error which will be not related to my change.